timestamp-batch/src/main/java/io/spring/configuration/TimestampBatchTaskConfiguration.java

```diff
@@ -50,18 +58,46 @@
 	@Autowired
 	private TimestampBatchTaskProperties config;
 
+	/**
+	 * Override default transaction isolation level 'ISOLATION_REPEATABLE_READ' which Oracle does not
+	 * support.
+	 */
+	@Configuration
+	@ConditionalOnProperty(value = "spring.datasource.driver", havingValue = "oracle.jdbc.OracleDriver")
+	static class OracleBatchConfig {
+		@Bean
+		BatchConfigurer oracleBatchConfigurer(DataSource dataSource) {
+			return new DefaultBatchConfigurer() {
+				@Override
+				public JobRepository getJobRepository() {
+					JobRepositoryFactoryBean factoryBean = new JobRepositoryFactoryBean();
+					factoryBean.setDatabaseType("ORACLE");
+					factoryBean.setDataSource(dataSource);
+					factoryBean.setTransactionManager(getTransactionManager());
+					factoryBean.setIsolationLevelForCreate("ISOLATION_READ_COMMITTED");
+					try {
+						return factoryBean.getObject();
+					}
+					catch (Exception e) {
+						throw new BeanCreationException(e.getMessage(), e);
```

Comment:
Wdyt about adding some context here such as `"Unable to override default transaction isolation level for Oracle due to: " + e.getMesage() ?